src/files.js

```diff
@@ -980,5 +984,394 @@ module.exports = (common) => {
         )
       })
     })
+
+    describe('.mkdir', function () {
+      it('make directory on root', (done) => {
+        if (!withGo) {
+          console.log('Only supported by go-ipfs yet')
```

Comment:
`Not supported in js-ipfs yet` would be better since those logs will only be printed when running js-ipfs tests.

Reply:
Sure!